test/base/module/bulkRequest/BulkRequest.deleteRecords.test.js

```diff
@@ -0,0 +1,16 @@
+import Auth from '../../../../src/node/authentication/Auth';
+import Connection from '../../../../src/node/connection/Connection';
+import BulkRequest from '../../../../src/base/module/bulkRequest/BulkRequest';
+import {USERNAME, PASSWORD, DOMAIN} from './common';
+
+describe('Checking BulkRequest.deleteRecords function', () => {
+  it('can be called without param', () => {
+    const auth = new Auth();
+    auth.setPasswordAuth({username: USERNAME, password: PASSWORD});
+    const connection = new Connection({auth, domain: DOMAIN});
+    const bulkRequest = new BulkRequest({connection});
+    expect(bulkRequest).toBeInstanceOf(BulkRequest);
+    const result = bulkRequest.deleteRecords();
```

Comment:
const result = bulkRequest.deleteRecords();
expect(result).toBeInstanceOf(BulkRequest);
=>
bulkRequest.deleteRecords();
//expect more after deleteRecords without param called an array of empty delete requests is added with empty/undefined records